prov/verbs/src/verbs_info.c

```diff
@@ -447,9 +474,20 @@ static int fi_ibv_get_device_attrs(struct ibv_context *ctx,
 		return -errno;
 	}
 
-	info->ep_attr->max_msg_size 		= port_attr.max_msg_sz;
-	info->ep_attr->max_order_raw_size 	= port_attr.max_msg_sz;
-	info->ep_attr->max_order_waw_size	= port_attr.max_msg_sz;
+	mtu_bytes = fi_ibv_mtu_type_to_len(port_attr.active_mtu);
```

Comment:
Move into if statement below, change to dgram_size, and merge with MIN functions below.  This will avoid setting the value unless it's needed, and reduce number of MIN calls.

Reply:
Thanks! Good catch.
Btw, there is duplication. Let's reduce duplication here